couple weeks of updates here

QuantumApprentice opened the pull request:
same ol, same ol.
please check the ones I leave comments on :)

QuantumApprentice committed:
Update lcrepair.msg

QuantumApprentice committed:
Update lcjunk.msg

QuantumApprentice committed:
Update pipboy.msg

QuantumApprentice committed:
Update lcdrugmr.msg

QuantumApprentice committed:
Update lcchlddg.msg

QuantumApprentice committed:
Update lcsteplt.msg

QuantumApprentice committed:
Update lcclgary.msg

QuantumApprentice committed:
what is a "flashman"?

{270}{}{You think it's a "small retail outlet". But think about it: how can a random flashman have so many goods to sell?
not a clue how to read this, maybe "salesman"?

QuantumApprentice committed:
Update lcclxa.msg

QuantumApprentice committed:
Update lcclbos.msg

QuantumApprentice committed:
Update lchtlcrv.msg

{115}{}{Guess I can't talk down the price. I'll just pay and take the encoder to the factory.}
Changed this line to reference talking (speech skill check?) but still sound like colloquial english.
Hope it sounds good :)

QuantumApprentice committed:
Merge branch 'KeyboardGecko:main' into main

QuantumApprentice committed:
Update lcclwrka.msg

QuantumApprentice committed:
Update lcclwrkc.msg

QuantumApprentice committed:
Update lcclwrkb.msg

QuantumApprentice committed:
Update lcmrkgun.msg

I made this change because the original sentence was a bit unnatural, I think this one works better, but I'll happily defer to your opinion :)

QuantumApprentice committed:
Update wclifea.msg

QuantumApprentice committed:
Update wcmedic.msg

QuantumApprentice committed:
please check this one

This line: "I will be the switch that turns the electricity off. Who else but me!" sounds like it was intended as a pun or something, but doesn't make much sense the way its worded.
I swapped it with a sentence that does make sense, but it lacks the pun element...if there was one?
Please check on this one :)

QuantumApprentice committed:
what does "knocked out of the rut" mean?

"knocked out of the rut" isn't a phrase I'm familiar with.  I figured 1920s style gangster talk was the intent, so I replaced it with a 1920s style gangster phrase.  Does this work? or is it too much?

QuantumApprentice committed:
Update nceldbos.msg

QuantumApprentice committed:
Update scnisgvn.msg

QuantumApprentice committed:
Update echsman.msg

QuantumApprentice committed:
"landlord" vs "boss" vs "owner"

Changed  {133} "for a chance of a contact." to something more familiar in English, but I'm not sure I have the pronoun right.  I went with "landlord", but it could be "boss", "owner", etc...
Any sound better to you?

QuantumApprentice committed:
Update pro_item.msg

QuantumApprentice committed:
Update echarsln.msg

QuantumApprentice committed:
Update hsventil.msg

QuantumApprentice committed:
Update wcpost.msg

QuantumApprentice committed:
"chat and swing with fists"?

{112} changed to "chat and swing our fists".  Can't tell if this is meant to reference fist-fighting? or something else?
Also changed "militaries" to "soldiers", the conversations make more sense with this, unless the farmer was intentionally using the wrong word?

QuantumApprentice committed:
double check these please :)

{118}{}{So, now you have a box for a lifetime} 
sounds like a reference to marriage?
changed "drop a fiancee on my neck!" to "drop a fiancee on my lap!", could be "hang a fiancee around my neck!" if that works better?

QuantumApprentice committed:
Update echarsln.msg

KeyboardGecko reviewed (approved):
I think it's something like "fly-by-night trader" or stray merchant, smth like that. Coming out of nowhere all of a sudden.

QuantumApprentice commented:
well...hmm...I can't think of a good English word for that.
I'll try and think about this, see if I come up with anything.

QuantumApprentice committed:
Update wcspy.msg

QuantumApprentice committed:
Update vcdmbwkc.msg

QuantumApprentice committed:
Update pro_item.msg

QuantumApprentice committed:
not sure about this one

"{115}{}{Anyway, there were two brothers."... " One of them Tom was always helping the closest... or even distant."
I'm not sure exactly what he's referring to here.  Swapped closest for poorest, and distant for richest, but I'm not sure if that's equivalent to the original.
Any ideas?
Also swapped "sued" for "conned", seemed like it fit the character better, but what do you think?

QuantumApprentice committed:
Update ecbobby.msg

QuantumApprentice committed:
Update wcbarter.msg

QuantumApprentice committed:
this line has a buggy reference! able to fix?????

{151}{}{Look at the report on current inmates.}
When I clicked this line in the dialogue (part 18 1:01:40) it sent me to an "Error" message.
I checked the script and this line appears to be called from NodeAnswer004, and it points to NodeAnswer004b.
NodeAnswer004b points to lines 153, 154, and 152, of which only 152 exists, and the response appears to be based on a quest variable.

Are there are couple of lines missing here? Should we fill them in?

QuantumApprentice committed:
Update tstrmrd.msg

QuantumApprentice committed:
Update pipboy.msg

QuantumApprentice committed:
Update scnisgvn.msg

QuantumApprentice committed:
Update scforsmn.msg

QuantumApprentice committed:
Update vbccmndr.msg

QuantumApprentice committed:
Update occitzn.msg

QuantumApprentice committed:
Update pro_item.msg

QuantumApprentice committed:
thoughts on this line?

{19111}{}{But I believe - it's all for this holy crusade!}
Swapped "best cause" for "holy crusade"...what do you think?

QuantumApprentice committed:
Update pipboy.msg

QuantumApprentice committed:
Update pipboy.msg

QuantumApprentice committed:
Update vcoplghl.msg

QuantumApprentice committed:
Merge branch 'KeyboardGecko:main' into main

QuantumApprentice commented:
> I think it's something like "fly-by-night trader" or stray merchant, smth like that. Coming out of nowhere all of a sudden.

ok, here's a couple of ideas: 
"used-car salesmen" is sometimes used pejoratively to mean somebody who tries to pass off a piece of junk as a car.

"carpet-bagger" oxford dictionary defines this as "a person perceived as an unscrupulous opportunist."

"high pressure salesman" is just what it sounds like, somebody who has a very pushy sales pitch, aggressive and manipulative

not sure if these work or not tho

QuantumApprentice committed:
Update vcrgbos.msg

QuantumApprentice committed:
Update bcjastin.msg

QuantumApprentice committed:
Update bcvillym.msg

QuantumApprentice committed:
Is this ok?

I added this line to make it more obvious where to look for the next clue.  
I spent a couple of weeks trying to figure out where to find the next missing brahmin clue and ended up looking it up in the guide just because I didn't think to click on any of the fence art. 
My hope is that new players won't be as lost as I was.

QuantumApprentice committed:
Update bcvillym.msg

QuantumApprentice committed:
Update bcvillym.msg

QuantumApprentice committed:
Update bcjastin.msg

swapped out one of the lines because "to not make you nervous any further" doesn't make much sense and is a little awkward to say.

QuantumApprentice committed:
Update bcslave.msg

QuantumApprentice committed:
Update pro_item.msg

QuantumApprentice committed:
Update rcnlsrdb.msg

QuantumApprentice committed:
Update rcatmman.msg

QuantumApprentice committed:
Update rcworkb.msg

QuantumApprentice committed:
Update wccrvbig.msg

QuantumApprentice committed:
Update vcdmbbos.msg

QuantumApprentice committed:
Update vhmer.msg

QuantumApprentice committed:
archivist's missing daughter quest lines

I changed this line:
"In the "Hard Rock Cafe" no one has seen her, in "3 Card Monte" she didn't appear for a long time,"
to:
"No one has seen her in the "Hard Rock Cafe", she's been missing from her job at "3 Card Monte" for a long time"
This sounds better to me, and I'm pretty sure she had a job at the casino, but it might have been the Hard Rock Cafe.  Do you remember her details?

QuantumApprentice committed:
Update vhdevis.msg

QuantumApprentice committed:
Update vctibbet.msg

QuantumApprentice committed:
please check on this one :)

{59301}{}{Holodisk with a recording you made while talking to Hal Wallace about getting a job for the freaks. Police will find it interesting.}
and:
{59501}{}{A holodisk with a recording you made while talking to Hal Wallace about getting a job for Bob Neil. Direct evidence.}

Not sure if "getting a job on freaks" translates correctly to "getting a job for the freaks", and a similar problem with the "Bob Neil" line.
Was that conversation about getting jobs for the characters? or plotting against them? or previously working with them? (ie the drug conversations with Sam Phillips)

QuantumApprentice committed:
Update vcdrgdlr.msg

QuantumApprentice committed:
"art of a bloody fight" vs "art of pugilism"

There's a guy in the Klamath bar who will teach you "Pugilism, the manly art of hand-to-hand combat." (John Sullivan)
I figure this might be a good reference to make?

QuantumApprentice committed:
Update vcrgtrnr.msg

QuantumApprentice committed:
Update vcslvsmk.msg

QuantumApprentice committed:
Update scstslvx.msg

QuantumApprentice committed:
Update scstslvx.msg

QuantumApprentice committed:
Update vcbar.msg

QuantumApprentice committed:
Update vcflick.msg

QuantumApprentice committed:
Update pro_item.msg

QuantumApprentice committed:
Update rcmerlin.msg

QuantumApprentice committed:
"blinds your eyes" vs "leaves you blind"?

{162} ... "Sometimes they keep you sober when success blinds your eyes."
vs:
"success leaves you blind."

I'm not sure what the original translation is, I can't think of anything that sounds similar in English.
Any thoughts?  Maybe it means overconfident? or something similar?

QuantumApprentice committed:
Update vcflick.msg

QuantumApprentice committed:
Update vchlputb.msg

QuantumApprentice committed:
Update vchlputb.msg

QuantumApprentice committed:
Update vcmertlk.msg

sorry, made a lot of changes in this one, hopefully they all sound good, let me know if they don't :)

QuantumApprentice committed:
Update vcbar.msg

QuantumApprentice committed:
Update vcmntptx.msg

QuantumApprentice committed:
help on this one?

QuantumApprentice committed:
Update vcmntptx.msg

QuantumApprentice committed:
Update vhhall.msg

QuantumApprentice committed:
Update ecignbob.msg

QuantumApprentice committed:
are "well"  & "tell"  supposed to rhyme here?

{242}{}{Well, well, well! What do you have to tell?}
changed it to "tell me" because that's more common english, but if it's intended to rhyme then the original is ok.

{224}{}{Hal Wallace?! This upstanding man was involved in criminal activities? You understand that you are responsible for your words right now?}
Replace "man" with citizen as "upstanding citizen" is more common in police procedural tv shows, but "are responsible for your words" is a little more awkward.  Usually the officer asks if the person is 'willing to testify before a court', but I don't think this game has anything like that in this quest, so I can't assume the player will ever be called on to testify.  I went with "will be held to account" because it matches the original meaning and is more common, but what do you think?

QuantumApprentice committed:
Update pro_item.msg

QuantumApprentice committed:
"redistribution of shares"?

"There was some confusion at City Hall about the redistribution of shares"
I'm not exactly sure what "shares" he's talking about here, so I left it alone.
Is he referring to some specific trade I'm missing?

QuantumApprentice committed:
Update pro_item.msg

QuantumApprentice committed:
confusing diologue options?

I ran into dialogue in this order:
////////
{182}{}{It was some dealer... I don't remember who exactly, there are so many of them lately, and there are always new faces in the business. Looks like the police of Clean City never rest.}
//one of the dialogue options
{175}{}{Why would they need that?}
//the response
{183}{}{First, not everyone likes to live beside crazy idiots. Second, the Clean City tries to establish a monopoly on everything. Perhaps, they are trying to take control of the laboratory, too...}
////////
given the context I changed the line to "Why would they never rest?", but I'm not entirely sure I understand the reference.

QuantumApprentice committed:
"tunable to a note re" ?

I swapped this line out for "a high e", but it could also be "a low e" ?
Six string guitars have 2 "e" notes in standard tuning:
The high "e" is usually 9 or 10 guage and the low "e" is usually 40 to 45 guage.
Does the original specify any of this?

QuantumApprentice committed:
Update pro_item.msg

QuantumApprentice committed:
bunch of fixes...

bunch of changes here, I added this line:
{130}..."You can find him at the Hard Rock Cafe."
because when I went through dialogue there wasn't any option to ask where Orris was the first time through,
so I got lost for a bit before coming back and asking a second time.

changed "more convenient for them to lick asses" 
to
"more convenient for them to kiss the asses"
pretty sure this is what's meant...unless "lick ass" means something else in the original script?

QuantumApprentice committed:
Update pro_item.msg

QuantumApprentice committed:
you scratch my back...

changed:
{158}{}{I don't know what the fuck are you talking about. Nobody does anything without personal gain in Vegas. You do something for me, I do something for you. Get it? So, maybe we can make a deal.}
to:
{158}{}{I don't know what the fuck you're talking about. Nobody does anything without personal gain in Vegas. You scratch my back, I scratch yours. Get it? So, maybe we can make a deal.}

I figured the "You scratch my back..."etc line sounded better and is a common colloquialism in the US. What do you think?

QuantumApprentice committed:
Update vckndma.msg

QuantumApprentice committed:
bunch of changes, may want to inspect

QuantumApprentice committed:
Update vihllist.msg

QuantumApprentice committed:
Update vccopzam.msg

QuantumApprentice committed:
Update pro_item.msg

QuantumApprentice committed:
Update vcbirda.msg

QuantumApprentice committed:
Update pro_item.msg

QuantumApprentice committed:
Update zcrcrvna.msg

QuantumApprentice committed:
malignent vs violent?

{219}{}{Dust storms are malignant. Knocks you off your feet.}
changed "malignant" to "violent" because malignant isn't usually used to describe something that "knocks you off your feet",
but more to describe diseases that can't be cured and are fatal.

QuantumApprentice committed:
Update vscrvblk.msg

QuantumApprentice committed:
Update ecbhuntr.msg

QuantumApprentice committed:
Update liseif.msg

QuantumApprentice committed:
"residents of paramount Wastes"?

I changed it to "kings of the Wasteland", but I'm not sure what the original intent was.
"paramount" is used out of order here, it could be "paramount residents of the Wastes", but even that comes across a bit clunky.
What do you think?

QuantumApprentice committed:
Update zcrcrvna.msg

QuantumApprentice committed:
Update wcbarter.msg

QuantumApprentice committed:
painted?

{250}{}{Hurry! One of the caravaneers needs medical help!}
{251}{}{Uh... [shakes his head] Another one was painted? It often happens here.}
changed "painted" to "beaten"...pretty sure that's what happened to the caravaneer this is about, but what does "painted" mean?

QuantumApprentice committed:
Update wccrvbob.msg

QuantumApprentice committed:
lot of little fixes here

QuantumApprentice committed:
Update wcpost.msg

QuantumApprentice committed:
Update ncstrmom.msg

QuantumApprentice committed:
Update wcprvtdi.msg

{154}{}{...And on top of that - abandoning your military unit. What was that talk about the "untarnished honor of the uniform"?}
//
I'm not familiar with what "untarnished honor of the uniform" is referring to, there's probably an equivalent phrase in english, but I can't think of it right now.

{150}{}{What would you know about honor, you desert hobo?! If it weren't for us, you all would have been slaves long ago. And what thanks do we get? These hagglers from the wastelands rip us off every time! So it's only fair to take what's ours, even by force.}
{151}{}{You did the right thing, letting it slip. Now back to the point.}
//
Line 151 is a dialogue response to line 150...but it doesn't make much sense
The reply is for selecting 151 is:
{160}{}{[Spits under your feet] You think some civilian rat can push me against the wall? Don't forget who's turf you're standing on!}
Any ideas what this is supposed to be? Kind of confusing to me.

QuantumApprentice committed:
Update wcwescer.msg

QuantumApprentice committed:
Update vcctzbnd.msg

QuantumApprentice committed:
Update vcctzbnd.msg

QuantumApprentice committed:
check on this one?

I think this sounds good...but you know

QuantumApprentice committed:
Update zswall.msg

QuantumApprentice committed:
Update zsvtbns.msg

QuantumApprentice committed:
Update vccldvor.msg

QuantumApprentice committed:
Update pro_item.msg

QuantumApprentice committed:
Update pro_item.msg

QuantumApprentice committed:
Update vckndma.msg

QuantumApprentice committed:
Update vccldvor.msg

QuantumApprentice committed:
this one is confusing

{125}{}{Your boss gave me a civil job to snitch Phillips.}
I'm not sure what "civil" means exactly.
And the response is {126}{}{Oh, that changes things. ...
where the officer describes everything that happened so far...so I'm not sure what "snitch Phillips" means either.
Snitch definition:
1. steal.
"he snitched the umbrella for when he went fishing"
2. inform on someone.
"she wouldn't tell who snitched on me"

QuantumApprentice committed:
lot of changes here, please double check?

QuantumApprentice committed:
Update ssstbcmp.msg

QuantumApprentice committed:
Update ssstbcmp.msg

QuantumApprentice committed:
Update ssstbcmp.msg

QuantumApprentice committed:
Update pro_item.msg

QuantumApprentice committed:
Update vccrvwnd.msg

Line 14: 
{116}{}{Well, thank you. Was happy to see you.}

Is used as a response to Line 12:
{114}{}{Vocas. The guys just got back from Salt Lake City. You know, there's a great salt lake there. A mutated water lily grows there, it doesn't grow anywhere else. Twice a year it blooms and produces nuts. Nibble them, you won't regret it. They taste bitter, but they help to flush all kinds of harmful crap out of your body. In short, useful stuff.}

Changed it to:
{116}{}{Well, thank you. These look like tasty nuts.}
I think this keeps the dirty joke connotations the original seemed to have, but makes more sense in the dialogue.

QuantumApprentice committed:
Update vcbirda.msg

QuantumApprentice committed:
Update zcrcrvna.msg

QuantumApprentice committed:
Update vccafbob.msg

changed
"exploding butt on the toilet for a day, if you eat too much."
to
"exploding butt syndrome. You'll be on the toilet for a day if you eat too much."
hope it still captures the humor :)

QuantumApprentice committed:
Update pro_item.msg

QuantumApprentice committed:
Update vhfillps.msg

QuantumApprentice committed:
Update vccopzam.msg

QuantumApprentice committed:
Update vcrgbos.msg

QuantumApprentice committed:
Update lcsherif.msg

QuantumApprentice committed:
What tipped you off? 

{318}{}{Hmm. Interesting. What tipped you off? ...
This is in response to {316}{}{Yes! I want to be on your show!}
But this character had actually said {308}{}{The folks themselves step up onto "the podium". ...
in an earlier conversation node.
So either he tipped off the player himself (which would need a line to indicate that...possibly as a joke?) or this line might not reflect the original dialogue correctly.
For now I changed it to

QuantumApprentice committed:
Update pro_item.msg

QuantumApprentice committed:
Update pro_item.msg

QuantumApprentice committed:
Update pro_scen.msg

QuantumApprentice committed:
Update vcmerman.msg

QuantumApprentice committed:
Update wcrepair.msg

QuantumApprentice committed:
"don't even try to squeak"

{156}{}{Our guys won't let you get away with this bullshit. Shut the fuck up and don't even try to squeak about our brotherhood.
Is the "don't even try to squeak" part intentional?
I swapped it out for "don't even make a peep", because it seems pretty close and is more common.

QuantumApprentice committed:
"I didn't crawl from anywhere..."

{146}{}{I didn't crawl from anywhere... You know what, actually that is true to some extent.}
I think this might be funnier as:
"I didn't crawl from anywhere...but yeah it was a hole." or something like that.
What do you think?

QuantumApprentice committed:
Update bcgirl.msg

QuantumApprentice committed:
Update bcphill.msg

QuantumApprentice committed:
Update zcrcrvna.msg

QuantumApprentice committed:
Update zcdlzns.msg

QuantumApprentice committed:
{114}{}{We'll talk later.}

{114}{}{We'll talk later.}
changed to:
{114}{}{Let me ask you something else.}
part 21 of my playthrough I run into this line of dialogue being used to return to the main dialogue tree, so I swapped it out for something that made more sense.
Only problem is I'm not sure if this line is used somewhere else to end dialogue so...here's hoping.

QuantumApprentice committed:
Update nccrvtlk.msg

QuantumApprentice committed:
Update ncnobikr.msg

QuantumApprentice committed:
Update pro_item.msg

QuantumApprentice committed:
Update pipboy.msg

QuantumApprentice committed:
Update ncdlzns.msg

QuantumApprentice committed:
Update ncnobikr.msg

QuantumApprentice committed:
So, the bird now sings another song...

{586}{}{So, the bird now sings another song...}
changed to:
{586}{}{So, the bird is singing a new tune...}
but I'm not sure what the original intent of this phrase is.
Maybe "Changes its tune" instead?

QuantumApprentice committed:
Update bcphill.msg

QuantumApprentice committed:
{249}{}{How much longer?}

{249}{}{How much longer?}
This line is used as a segue between showing him (biker Phill) weapons to upgrade.  It doesn't make much sense as a response to most of the dialogue choices given, so I changed it to:
{249}{}{What else ya got?}
But I'm not sure if any of the other dialogue lines actually do correctly lead to "How much longer?"...so let's hope they don't.

QuantumApprentice committed:
Update bcgirl.msg

QuantumApprentice committed:
Update bcphill.msg

QuantumApprentice committed:
"prematures"? and 3 untranslated lines

{469}{}{You really need 'em? Hmm. You proved yourself well with these prematures in Reno and deserve our respect. So be it - I will share them with you. You're good kid.}
changed "these prematures" to "those problems"
same on {1469}
I have no clue what "prematures" is supposed to be, but it's pretty clear he's referring to the new reno caravan quest, so I changed it to "problems".  Does this work? Is there something else he's referring to?
and the female dialogue has 3 untranslated lines: {1530}, {1532}, {1533}
also changed
{216}{}{It's simple. I'm actually not so sure myself, so - wanna bet?}
to 
{216}{}{It's simple. But if you're so sure I can't do it - wanna bet?}
Thoughts on this change? seems close to the intent and makes more sense.

QuantumApprentice committed:
Update bcphill.msg

QuantumApprentice committed:
Update bisterm.msg

QuantumApprentice committed:
Update bisterm.msg

QuantumApprentice committed:
"pops his eyes frantically"?

{286}{}{Oh no... [pops his eyes frantically ..."
changed to
{286}{}{Oh no... [rubs his eyes frantically ..."
not sure what "pops his eyes" is supposed to refer to, so I replaced with "rubs his eyes",
but if it's supposed to be something else, please change to reflect

QuantumApprentice committed:
Update vcmertlk.msg

QuantumApprentice committed:
"Hands-out-of-ass"?

{173}{}{Hands-out-of-ass - that's about you!}
uh...not sure what this is supposed to be
maybe "Sitting on your hands"? since many of the other dialogue lines are about somebody being lazy?
or maybe "Head up your ass", ie "so dumb they have their had up their ass"?

QuantumApprentice committed:
does this one sound good?

{55201}{}{This is your personal sales license, issued by the manufacturer of the famous tonic beverage "Nuka-Cola".}
changed to
{55201}{}{This is your personal sales license, issued by the wasteland famous tonic beverage manufacturer, "Nuka-Cola".}
Made this change to make the description sound more like a sales pitch

QuantumApprentice committed:
Update vcrgboxc.msg

QuantumApprentice committed:
Update lcclscld.msg

QuantumApprentice committed:
Update lscolabx.msg
